Buildings/Controls/OBC/FDE/DOAS/EconMode.mo

```diff
@@ -0,0 +1,127 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block EconMode "This block calculates when economizer mode is active."
+  parameter Real econCooAdj(
```

Comment:
dTEcoThr. Threshold temperature difference between return air and outdoor air temperature above which economizer mode is enabled